### DISCUSSION TITLE
Unable to reapply using same ID

### DISCUSSION THREAD

**Discussion by kalpajee** [community/community#108606] [GitHub Education]
### Hi everyone,

When i trying to upload university ID as a proof there is a error message appearing "Your uploaded proof has already been used previously". what can i do now. I only have a digital copy of my ID

**Comment by Cizr** [comment]
check if you've submitted it previously. If not, contact customer support explaining the issue. If it has been used, review the platform's document reuse policy and, if allowed, ensure your digital copy meets format and size requirements. Contact support for guidance if needed.

**Reply by kalpajee** [reply]
I submitted it previously. what can i d now? i submitted the digital copy
